src/App.tsx

```diff
@@ -1,7 +1,36 @@
-import React from 'react';
+import { Component } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedKey: string;
+};
```

Comment:
let's make it nullable instead, more consistent
```suggestion
type State = {
  pressedKey: string | null;
};
```